package-lock.json

```diff
@@ -19364,16 +19364,6 @@
 						"yauzl": "^2.7.0"
 					},
 					"dependencies": {
-						"are-we-there-yet": {
```

Comment:
These changes were generated. They are unrelated to the implementation details of the component being tested, or the tests.